tests/transformers/tests/generation/test_utils.py

```diff
@@ -2328,7 +2328,7 @@ def test_generate_from_inputs_embeds_decoder_only(self):
             )
             self.assertListEqual(
                 outputs_from_embeds[:, inputs_embeds.shape[1] :].tolist(),
-                outputs_from_embeds_wo_ids[:, 1:].tolist(),
+                outputs_from_embeds_wo_ids.tolist(),
```

Comment:
This was an issue with the original test, as confirmed by disabling `adapt_transformers_to_gaudi`.

`outputs_from_embeds` contains the input tokens (first 3 tokens) and the output tokens (final 17 tokens).
`outputs_from_embeds_wo_ids` contains just the output tokens (17 tokens).

This comparison was getting rid of the first generated token in `outputs_from_embeds_wo_ids`, which appeared to be erroneous.

Reply:
https://github.com/huggingface/transformers/blob/277db238b70c5c8bb5a445b75becbfd79644db42/tests/generation/test_utils.py#L1511